apps/authentication-portal/src/main/webapp/basicauth.jsp

```diff
@@ -288,6 +292,12 @@
     <div class="ui divider hidden"></div>
 <% } %>
 
+<% if (isAdminSessionAdvisoryControlEnabledInTenant == true) { %>
+    <div class="ui warning message" data-testid="login-page-admin-session-advisory-banner">
```

Comment:
Shall we use `data-componentid` instead? we are moving away from `data-testid` therefore new code should not have `data-testid`.